syntax = "proto3";

import "proto/common.proto";

message SchedulerIntroduceRequest {
	string rpc_uri = 1;
	string client_rpc_uri = 2;
}

service SchedulerService {
	rpc SubmitJob (Job) returns (Empty) {}
	rpc Done (Task) returns (Empty) {}
	rpc Introduce (SchedulerIntroduceRequest) returns (Empty) {}
}